### PR TITLE
Rs 2022.10.09

### PR DESCRIPTION
**Pull request by rschlaikjer**
This is something beyond the imagination; a fantasy.

2022-10-09-140642_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 1]
```diff
@@ -317,47 +342,52 @@ It was a lightning-fast movement, almost as if it was of the utmost importance.
 [sha:23546edcd305ad726c2e848c1b841aadd0501133]{
 -- Page 273, Offset 18950.
 -- うん、ホントに大漁でした。
-Yeah, I've seriously hauled in a good catch.
+Yeah, I've landed a big catch alright.
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:345]: senpai is a big fish

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 2]
```diff
@@ -367,62 +397,66 @@ Senpai takes a spoon from the basket and begins to timidly eat the butter chicke
...
 -- Page 280, Offset 18969.
 -- そんな事を切り出してきた。
-And so she asks.
+She gets right to the point.
 }
 [sha:f920dd3605d095f17cc6aaeccca370545b727351]{
 -- Page 281, Offset 18970.
 -- 「いえ、怖いですよ。銃を持てば戦争は怖くなくなる、なんて話は聞いた事もないし。
-"No, I am scared. Who's ever heard of someone not afraid of war just because they have a gun?
+"No, I'm scared. It's not like anyone stops being afraid of war just because they have a gun, right?
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:453]: afraid in war just because

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 3]
```diff
@@ -942,17 +1026,21 @@ Senpai smiles in satisfaction and heads for the exit, for real this time.
 [sha:032079b0d803bc8491d6cf88fe71a36dba4102e3]{
 -- Page 328, Offset 19075.
 -- 「ありえませんね。呪いならともかく、人間でいう風邪を引く、なんてナンセンスです。死者に免疫機能はありませんから」
-"Impossible. Setting aside the eventuality of a curse, catching a human cold is utter nonsense. The Dead don't have immune systems, after all."
+// Surely not having an immune response makes you _more_ able to catch a cold but what do I know, I only took 5 years of bio
```
Comment (Avuxo) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:1029]: well, they don't have immune _responses_ so they ain't gettin a fever i guess

**Review by Avuxo — APPROVED** [review]
otsukare

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 4]
```diff
@@ -755,38 +814,51 @@ After all, death permeates the entire world.
...
-"But what your eyes see is different. To skip over the process, directly putting an end to things is to emancipate from the law of conservation of energy. It is the pinnacle of the meaning of death.
+// Possibly too liberal, but based on this snippet from an article about stands it seems like
+// 過程/結果 is used as cause/effect would be in EN
+// https://hungry-bear.net/kingcrimson_stand/
+// 過程を飛ばして物事を終わらせるのなら、それはエネルギー保存則に頼らない、究極の意味での死です。
+// which I think is consistent with other descriptions of his eyes? Literally 'skipping a process' makes it sound like you gotta
+// file paperwork to kill somebody
+"But what your eyes see is different.
+ Skipping the cause and jumping straight to the effect, forcefully putting an end to something, is to kill it in the truest sense.
+ It completely disregards the law of conservation of energy.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:844]: Yeah, 過程 as cause is totally fine, but I'm not sure I'd want effect in there together with it.

> If you can bring an end to something while ignoring the causality of that action, you would be disregarding the law of conservation of energy. That would be the death in the truest sense of the word.

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 5]
```diff
@@ -755,38 +814,51 @@ After all, death permeates the entire world.
...
 }
 [sha:2c412bb92c25e6878fc10d55b9aa358feae1d648]{
 -- Page 312, Offset 19042.
 -- ある司祭からの受け売りですが、<無|し>とはこの宇宙にいまだないもの。在ると想定する事で多くの事象を説明できるけれど、実在してはいけないものとされています」
-I heard this from a priest, but <nothingness|d e a t h   > is something that doesn't yet exist in our cosmos. Supposing its existence allows us to explain many phenomena, but it is seen as something that must not exist."
+// Death is dark matter, naruhodo (doesn't naruhodo at all)
+A priest once said to me that <nothingness|d e a t h    > is something that doesn't yet exist in our cosmos.
+ Supposing its existence allows us to explain many phenomena, but it is seen as something that must not exist."
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:851]: Presbyter (she's talking about Roa).

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 6]
```diff
@@ -755,38 +814,51 @@ After all, death permeates the entire world.
...
 }
 [sha:58dab170b0577ad9e359da4ccbe4d3fcf31d6098]{
 -- Page 313, Offset 19043.
 -- ふむふむ、と鹿爪らしく頷く。
-I solemnly nod in comprehension.
+I solemnly nod as though this all makes perfect sense.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:856]: What's the matter Shiki-boy?

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 7]
```diff
@@ -796,78 +868,88 @@ Senpai continues staring at me mercilessly.
...
+Could you dumb it down for me a bit?"
 }
 [sha:679f2a254e8217669f103ed9abc4f67633a36084]{
 -- Page 315, Offset 19047.
 -- 「つまり空想上のものです。ファンタジーですね。ありきたりのものどころか、どこにもないモノってコトです。
 // ???
-"In short, it is something above imagination: a fantasy. Though it is a commonplace concept, it does not exist anywhere.
+// :thinkies:
+"In short, it is something beyond the imagination; a fantasy.
+ It isn't something commonplace at all, but something that doesn't exist %{i}anywhere%{/i}.
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:879]: どころか feels like it didn't get encoded quite well enough.
> It's not just uncommon, in fact, it doesn't exist anywhere at all.

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 8]
```diff
@@ -796,78 +868,88 @@ Senpai continues staring at me mercilessly.
...
 }
 [sha:4e0bd1d6a2a021aafff960619ac18ef9185333d0]{
 -- Page 315, Offset 19048.
 -- ……まあレアケースなのは確かですが、神話にはそういった魔眼の伝承があるので、過去に何人かの<保有者|ホルダー>はいたんでしょうけど」
-...Well, your eyes certainly are a rare case, but legends of those Mystic Eyes have passed down, so there must have been multiple holders in the past."
+Though, I suppose there are some exceptions...
+ There are a handful of legends that refer to Mystic Eyes like yours,
+ so there must have been multiple holders in the past."
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:886]: 神話 here really does refer to mythology. She's referring to Balor, who Nasu has said had the eyes in the past.

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 9]
```diff
@@ -942,17 +1026,21 @@ Senpai smiles in satisfaction and heads for the exit, for real this time.
 [sha:032079b0d803bc8491d6cf88fe71a36dba4102e3]{
 -- Page 328, Offset 19075.
 -- 「ありえませんね。呪いならともかく、人間でいう風邪を引く、なんてナンセンスです。死者に免疫機能はありませんから」
-"Impossible. Setting aside the eventuality of a curse, catching a human cold is utter nonsense. The Dead don't have immune systems, after all."
+// Surely not having an immune response makes you _more_ able to catch a cold but what do I know, I only took 5 years of bio
+"Impossible.
+ A curse is one thing, but catching a human cold is utter nonsense.
+ The Dead don't have immune systems, after all."
```
Comment (Petrikow) [inline script/Ciel/Day 8/08_01_CIEL08_3B.txt:1032]: 死者 here is just generic 'dead people'

Add that in the comment, too.

**Review by Petrikow — COMMENTED** [review]
fuck curry

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 10]
```diff
@@ -81,67 +91,71 @@ But sadly, I don't have the means to eat out this month.
 [sha:3b6ae7533bb32a4475ef9c6215eead1caa35bcfb]{
 -- Page 253, Offset 18903.
 -- 先輩が“いい店”と言うぐらいだし、目を疑うほどの値段設定のレストランかもしれない。
-Senpai talked about 'a nice place', so it could be a fancy restaurant with eye-popping prices.
+Senpai said that it was a 'great place', so it could be one of those fancy restaurants with eye-popping prices.
```
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 8/08_01_CIEL08_3B.txt:94]: Seems Akiha doesn't give much of an allowance to her brother

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 11]
```diff
@@ -81,67 +91,71 @@ But sadly, I don't have the means to eat out this month.
...
 }
 [sha:7b865af2f5b9db6232c49fa5faa908d953fd8572]{
 -- Page 253, Offset 18904.
 -- ここは断腸の思いで断るのができる男の対応である。
-Such is the reaction of a virtuous man, to sorrowfully turn down her offer.
+Though it breaks my heart, I must decline her offer.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 8/08_01_CIEL08_3B.txt:99]: "This is the response of a man who can decline despite his broken heart" just felt a bit overly wordy / doesn't add much imo

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/Ciel/Day 8/08_01_CIEL08_3B.txt:345]: Is this a comment or a suggestion?

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 12]
```diff
@@ -226,17 +245,19 @@ Now that I've said this much, she's got to deliver the usual punchline.
...
 -- Page 266, Offset 18932.
 -- つまり、女子高生の一日の平均エネルギー摂取量的なものを考慮して我慢している、と。
-In other words, she's restraining herself and adhering to the average caloric intake of a highschool girl.
+In other words, she's restraining herself and adhering to the average caloric intake of an ordinary high school girl.
 }
 [sha:d97e0fe04906baee6ff5f6aef581b8b83a4d6a33]{
 -- Page 267, Offset 18933.
 -- バターチキンカレーはたいへん美味しかった。
+// Not sponsored
 The butter chicken was finger-licking good.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 8/08_01_CIEL08_3B.txt:254]: Yeahhh I debated cutting it as well. If you also did a double-take I might opt for sans meme

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt [hunk 13]
```diff
@@ -276,38 +299,40 @@ With her staring like that, I can't savor the curry I've just scooped into my sp
...
 -- わーい、と子供のようにスプーンにパクつくシエル先輩。
-With a childish 'yay~', Ciel-senpai swallows down the curry on the spoon.
+With childish glee, Ciel-senpai wolfs down the curry that was on the spoon.
 }
 [sha:84ece4ba9d62ac82048eaed2246d1ebec107f9df]{
 -- Page 271, Offset 18947.
 -- 一瞬の、待ったなしの早業だった。
-It was a lightning-fast movement, almost as if it was of the utmost importance.
+It happened in the blink of an eye,
+ as though she was terrified the chance would disappear.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 8/08_01_CIEL08_3B.txt:330]: I will grudgingly go against years of high school conditioning and use `like`.
I can shorten it, I was just trying to encode what I interpreted as the implication of 待ったなし

File: script/Ciel/Day 8/08_01_CIEL08_3B.txt (hunk 1 above)
Comment (Avuxo) [reply script/Ciel/Day 8/08_01_CIEL08_3B.txt:343]: the old version of the line. its my favorite mis-tl
